common/societies/wc_societies.txt

```diff
@@ -661,6 +661,7 @@ twilights_hammer_cult = {
 		}
 		decisions = {
 			develop_mutation
+#			forge_hammer_of_the_twilight
```

Comment:
Delete this or move to another branch

Reply:
commenting out is not enough? ok, I just thought it would be easier once sej wants to merge old gods, too. But sure, I can delete it

Reply:
zumbak said, comment out is enough :)